indy-wrapper/src/main/java/nl/quintor/studybits/indy/wrapper/Prover.java

```diff
@@ -64,6 +65,99 @@ public void init(String masterSecretName) throws IndyException, ExecutionExcepti
         return Anoncreds.proverStoreClaimOffer(wallet.getWallet(), claimOffer.toJSON());
     }
 
+    /**
+     * Proves the proofRequest using the stored claims
+     *
+     * @param proofRequest
+     * @param attributes This map is used to get the correct claim, if multiple referents are present, or to provide self-attested attributes
+     * @return
+     */
+    public CompletableFuture<Proof> proofRequestToProof(ProofRequest proofRequest, Map<String, String> attributes) throws JsonProcessingException, IndyException {
```

Comment:
`proofRequestToProof` is a bit vague. Can you think of a more explanatory function name?

Reply:
Done